requirements.txt

```diff
@@ -33,3 +33,4 @@ django-cors-headers==1.0.0
 
 # Zep 12 dependency
 django-uuslug==1.0.3
+watchdog==0.8.3 # use last non-bc-breaking version
```

Comment:
ça ressemble plus à un choix d'implémentation qu'à un bug, ça. si le répertoire écouté change, ça a de très fortes chances de rendre ton traitement caduque, non ? Et d'ailleurs ça me semble être un garde-fou plutôt sensé : si tu relances un traitement à chaque modif sans arrêter le traitement précédent c'est soit un "out of memory" soit le CPU qui va partir en sucette.